### DISCUSSION TITLE
Is there a planned support for Rollup ?

### DISCUSSION THREAD

**Discussion by Matwog** [vanilla-extract-css/vanilla-extract#279] [Q&A]
Hey Team, 
Thanks for vanilla-extract. 🎉 
The library makes sense for users who want to use CSS modules and stay close to native CSS but leverage type safety. From my understanding it looks like a promising css architecture to integrate with libraries using css design tokens. 

Is there a support planned for Rollup through a rollup plugin ?

**Comment by damianstasik** [comment]
https://github.com/seek-oss/vanilla-extract/issues/3#issuecomment-829726172
